docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -27,15 +27,23 @@ The `CREATE TABLE` statement defines a new table using Hive format.
 
 ```sql
 CREATE [ EXTERNAL ] TABLE [ IF NOT EXISTS ] table_identifier
-    [ ( col_name1[:] col_type1 [ COMMENT col_comment1 ], ... ) ]
+    [ ( col_name1[ : ] col_type1 [ COMMENT col_comment1 ], ... ) ]
     [ COMMENT table_comment ]
-    [ PARTITIONED BY ( col_name2[:] col_type2 [ COMMENT col_comment2 ], ... ) 
+    [ PARTITIONED BY ( col_name2[ : ] col_type2 [ COMMENT col_comment2 ], ... ) 
         | ( col_name1, col_name2, ... ) ]
     [ ROW FORMAT row_format ]
     [ STORED AS file_format ]
     [ LOCATION path ]
-    [ TBLPROPERTIES ( key1=val1, key2=val2, ... ) ]
+    [ TBLPROPERTIES ( key1 = val1, key2 = val2, ... ) ]
```

Comment:
Did you follow the existing format somewhere? I found a different format in https://github.com/apache/spark/blame/master/docs/sql-ref-syntax-ddl-create-table-hiveformat.md#L29-L38